SU2_CFD/include/variables/CEulerVariable.hpp

```diff
@@ -41,7 +41,25 @@ class CEulerVariable : public CFlowVariable {
 public:
   static constexpr size_t MAXNVAR = 12;
 
+  template <class IndexType>
```

Comment:
Oh right, because in CVariable everything is unsigned long, but in the numerics unsigned short, and so this way we keep Mr LGTM happy with narrow/wide comparisons.